src/parser/jobs/ast/changelog.tsx

```diff
@@ -1,10 +1,21 @@
+import {DataLink} from 'components/ui/DbLink'
 import CONTRIBUTORS from 'data/CONTRIBUTORS'
 import React from 'react'
 
 export const changelog = [
+	{
+		date: new Date('2024-08-24'),
+		Changes: () => <>Added <DataLink action={'LIGHTSPEED'} /> and <DataLink action={'SWIFTCAST'} /> to defensives. </>,
```

Comment:
Probably worth a mention that we're fixing the prepull display bugs here too

Reply:
no longer applicable for this PR